README.adoc

```diff
@@ -1,111 +1,240 @@
-= Database Update
+// Do not edit this file (e.g. go instead to src/main/asciidoc)
 
-How to write the code of your app to allow doing A/B testing or/and continuous deployment.
+:repo_url: https://github.com/marcingrzejszczak/database-update
 
-== Assumptions
+= Zero Downtime Deployment Database Issues
 
-In order for the process to be relatively simple the following assumptions need to take place
-while developing the app:
+{repo_url}
```

Comment:
Maybe move this to a second paragraph:

If you want to work though the code samples below, you will find everything you need in {repo_url}[GitHub].

Or something.